### PR TITLE
[FIX] sales: fix typo

### PR DESCRIPTION
**Pull request by alpa-odoo**
critarias -> criterias

### CODE REVIEW

File: content/applications/sales/sales/products_prices/prices/pricing.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ How to adapt your prices to your customers and apply discounts
 
 Odoo has a powerful pricelist feature to support a pricing strategy tailored to your business.
 A pricelist is a list of prices or price rules that Odoo searches to determine the suggested price.
-You can set several critarias to use a specific price: periods, min. sold quantity (meet a minimum order quantity and get a price break), etc.
+You can set several criterias to use a specific price: periods, min. sold quantity (meet a minimum order quantity and get a price break), etc.
```
Comment (meng-odoo) [inline content/applications/sales/sales/products_prices/prices/pricing.rst:7]:
```suggestion
You can set several criteria to use a specific price: periods, min. sold quantity (meet a minimum order quantity and get a price break), etc.
```

**Review by meng-odoo — CHANGES_REQUESTED** [review]
@alpa-odoo Hi! Thanks for finding this typo. The correct word is actually "criteria" (it's already plural without the "s" at the end). Let's just fix that tiny detail before approving :)

**Review by meng-odoo — APPROVED** [review]
Hi @alpa-odoo Thanks for making that change! Everything looks good. I'll move this to the next stage in the review process.

**Review by Feyensv — CHANGES_REQUESTED** [review]
Please squash the commits into one commit with a clear title (like the PR one) :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**